src/metabase/models/recent_views.clj

```diff
@@ -133,7 +133,7 @@
   [:and {:registry {::pc [:map
                           [:id [:or [:int {:min 1}] [:= "root"]]]
                           [:name :string]
-                          [:authority_level [:enum :official "official" nil]]]}}
+                          [:authority_level [:enum "official" nil]]]}}
```

Comment:
Guessing you saw [my comment](https://metaboat.slack.com/archives/C064EB1UE5P/p1715966517900699?thread_ts=1715959326.862959&cid=C064EB1UE5P) with my diagnosis of the error we were seeing on Friday but linking it just in case. Also the subsequent PR: https://github.com/metabase/metabase/pull/42844

Are we certain that `"official"` is _always_ a string? I think when you select `:authority_level` straight from `collection` it'll be automatically converted to a keyword via [this transformation](https://github.com/metabase/metabase/blob/ac906b13bfa1cb02198f689c68005c5937a04026/src/metabase/models/collection.clj#L102). But that breaks when a query aliases the column. So there's a chance it'll be a keyword if it's ever selected but not aliased.

Asked about the underlying t2 issue here: https://metaboat.slack.com/archives/CKZEMT1MJ/p1716234813449159

Reply:
I make it a string now in the get-parent-coll function by passing it through `#(some-> % name)`.